src/domain/landingPage/landingPageHero/LandingPageHero.tsx

```diff
@@ -32,6 +34,81 @@ const LandingPageHero: React.FC<Props> = ({ landingPage }) => {
     window.open(buttonUrl || '', '_self');
   };
 
+  // We want to have event padding for the text wrapper. By using pure CSS we can set
+  // max-width for the wrapper but when the text is longer that max-width the text
+  // will take space of max-width and there might be wider right padding.
+  // So we need to calculate the max-width of each text row and set max-width of
+  // text wrapper to same
+  const calculatedTextWrapperWidth = React.useMemo(() => {
```

Comment:
By the way, could this be extracted as a hook from here? To me this seems pretty long function in this component

Reply:
what do you think?

Reply:
You have a valid point here. It's not even useful here. Refactored